src/cim/components/InfraEnv/InfraEnvAgentTable.tsx

```diff
@@ -215,7 +231,7 @@ const InfraEnvAgentTable: React.FC<InfraEnvAgentTableProps> = ({
                 }}
               />
             ) : (
-              <HostsTableEmptyState setDiscoveryHintModalOpen={setDiscoveryHintModalOpen} />
+              <HostsTableEmptyState setDiscoveryHintModalOpen={setDiscoveryHintModalOpen} t={t} />
```

Comment:
you should not pass `t` to a component. Use hook to get the `t` in `HostsTableEmptyState`